### PR TITLE
Update to Docusaurous 3.0.0

### PR DESCRIPTION
**Pull request by dereknola**
## Changes
- Bump to new Docusaurous v3, largely this change concerns going from MDXv1 to MDXv3. As such, raw <> and {} needed to be converted (usually escaped or replaced with direct html code). See [Upgrade Guide](https://docusaurus.io/docs/migration/v3) if you want more info.
- Rework some tables to have a Default column for clarity
- Fix broken links

### CODE REVIEW

File: i18n/kr/docusaurus-plugin-content-docs/current/security/self-assessment.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This guide makes the assumption that K3s is running as a Systemd unit. Your inst
 **Remediation:**
 Run the below command (based on the file location on your system) on the
 control plane node.
-For example, chmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml
+For example, `chmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml``
```
Comment (VestigeJ) [inline i18n/kr/docusaurus-plugin-content-docs/current/security/self-assessment.md:45]: extra closing single quote?

File: i18n/zh/docusaurus-plugin-content-docs/current/security/self-assessment.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ This guide makes the assumption that K3s is running as a Systemd unit. Your inst
 **Remediation:**
 Run the below command (based on the file location on your system) on the
 control plane node.
-For example, chmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml
+For example, `chmod 644 /etc/kubernetes/manifests/kube-apiserver.yaml``
```
Comment (VestigeJ) [inline i18n/zh/docusaurus-plugin-content-docs/current/security/self-assessment.md:49]: Extra closing single quote?

**Review by VestigeJ — COMMENTED** [review]
I think just the two commented extra single quotes and then I can approve.

**Review by VestigeJ — APPROVED** [review]
Nooice

**Review by brandond — APPROVED** [review]

**Pull request merged**